wallet-user/js-example/readme.md

```diff
@@ -1,19 +1,19 @@
-# Js Example 
-This is a Js script to send BSC transactions with 0 gas price through MegaFuel Paymaster.
+# Js Example
+This is a Js script to send BSC transactions with 0 gas price through Megafuel Paymaster.
 
 ## Quick Start
 
-If the example is performed on BSC testnet, please ensure you have some test ERC20 on BSC testnet. (You can get some 
-from the official faucet).
+The example is performed on BSC testnet, please ensure you have some test ERC20 on BSC testnet. (You can get some
+from the official faucet)
 
 1. Install the dependency.
     ```shell
     $ npm install
     ```
 
 2. Edit the script.
```

Comment:
`Edit the script` --> `Configure the .env`

Reply:
fixed